2019/new_day_generator.sh

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 
-if [ "$1" != "-d" ]; then
-  usage;
-fi
+set -eu
 
-echo "Generating day$2"
+# What about $1?
 
```

Comment:
I was previously making a check that $1 was set as -d flag with the thought of extending this to include -y for future years.  If this is removed from the shell script, then could just use newDay = "$1" instead of newDay = "$2"

Reply:
But then you'd have to have an awkward conversation with all the users of the script - are you up to that?